gap/base/methods.gi

```diff
@@ -0,0 +1,90 @@
+#############################################################################
+##
+##  This file is part of recog, a package for the GAP computer algebra system
+##  which provides a collection of methods for the constructive recognition
+##  of groups.
+##
+##  This files's authors include Sergio Siccha.
+##
+##  Copyright of recog belongs to its developers whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
+##
+##  SPDX-License-Identifier: GPL-3.0-or-later
+##
+##
+##  Implementation of recog methods
+##
+#############################################################################
+
+InstallMethod(CallFuncList, [IsRecogMethod, IsList],
+{f, args} -> CallFuncList(f!.func, args));
+
```

Comment:
Do we really want this? I think you are using `CallRecogMethod` explicitly after all, and I think this is indeed better -- it is easy to grep for `CallRecogMethod` and it is then possible to track what happens. But if one replace `CallRecogMethod(meth, val)` by `meth(val)` then this is rather intransparent, and may be very confusing, esp. to people who don't know that one can overload function call syntax in GAP (i.e., >99.9% of them)
```suggestion
```

Reply:
I agree, we should probably drop `meth(val)`.

Reply:
Makes sense. I've removed this.